packages/commonwealth/client/scripts/views/pages/CommunityManagement/AdminsAndModerators/UpgradeRolesForm/UpgradeRolesForm.tsx

```diff
@@ -31,6 +37,33 @@ export const UpgradeRolesForm = ({
     { id: 2, checked: false },
   ]);
 
+  let upgradedUser;
+  let newRoleToBeUpgraded;
```

Comment:
lets not use these, maybe use react state or ref. Since these values will be cleared when the component re-renders.